packages/block-library/src/paragraph/index.js

```diff
@@ -65,6 +65,10 @@ const schema = {
 	customFontSize: {
 		type: 'number',
 	},
+	direction: {
+		type: 'string',
+		enum: [ 'ltr', 'rtl' ],
```

Comment:
do we support this `enum` property?

Reply:
Not yet 😉

Reply:
Well, to clarify, it's supported in the server-side validation, but isn't ported to the client. Idea with adding here now is it "just works" when ported.

Reply:
Related: https://github.com/WordPress/WordPress/blob/da7a80d67fea29c2badfc538bfc01c8a585f0cbe/wp-includes/rest-api.php#L1120-L1125

Reply:
There's probably a lot of block attributes where this could be considered. `align` for instance

Reply:
Related: #10635, https://github.com/WordPress/gutenberg/pull/9829#discussion_r225208672